source/PlayerInfo.cpp

```diff
@@ -177,8 +179,10 @@ void PlayerInfo::Load(const string &path)
 		{
 			for(const DataNode &grand : child)
 				if(grand.Size() >= 2)
-					reputationChanges.emplace_back(
-						GameData::Governments().Get(grand.Token(0)), grand.Value(1));
+					// reputationChanges.emplace_back(
+					// 	GameData::Governments().Get(grand.Token(0)), grand.Value(1));
+					GameData::Governments().Get(grand.Token(0))->
+						SetReputation(grand.Value(1));
```

Comment:
The original author of the code here had a reason to apply the reputation Changes later (in ApplyChanges). I'm not sure why, but I guess there was a good reason to delay applying those changes. Do you have an idea on why this applying of the changes was delayed?

And is there a reason not to do this? What if we just keep applying those reputationChanges later (because then they also overwrite any reputationChanges that were set from the conditions)?

Reply:
That still worries me. I can't see the logic of it, thus I may have missed something important. Unless:
- it was done because the mechanism for deferred-application was made and tested, so doing it that way felt neat and tidy
- or it mattered before the change to derived conditions, perhaps making sure condition-reputation was freshly updated from game-reputation, if the duplicator hadn't run

Could some of the conditions that get deferred-applied be operations to modify conditions? Seems strange to have any of those at load-time though. At least the reputation ones are just listed as absolute.

Reason not to do this? Specifically so old primary-condition-reputations take precedence, because they'll only exist if it's a savegame from the old system, and if any mission has set them they'll be more 'correct' than the gamedata-stored ones.
- otherwise if a mission changes your reputation (such as ceasing hostilities with Republic) before applying this fix, that change will be lost and you'll be forever hated by the republic

Reply:
> * or it mattered before the change to derived conditions, perhaps making sure condition-reputation was freshly updated from game-reputation, if the duplicator hadn't run

Yes, or maybe this was done because:
- other government data needed to be loaded before applying the reputations.
- mission loading somehow affects reputations, thus the actual reputations need to be set after loading the missions
- the default (standard) reputations from the data-files are loaded late in the process, causing the reputations from the savegame to be overwritten by the standard reputations from the data files.

ES uses "delayed loading" at more locations, and usually there was a very good reason for it due to interaction with other data or missing data. The reason here might no longer be valid, but we need to be really sure that such reason is no longer valid before I would be okay with this change. Just assuming that it will be no problem is not okay here, especially since assumptions (by me) on how loading works is what got us into this bug-fixing in the first place.